File: src/main/java/org/example/bankramenserver/domain/transaction/exception/TransactionNotFoundException.java
package org.example.bankramenserver.domain.transaction.exception;

import org.example.bankramenserver.global.error.exception.ErrorCode;
import org.example.bankramenserver.global.error.exception.GlobalException;

public class TransactionNotFoundException extends GlobalException {

    public static final GlobalException EXCEPTION = new TransactionNotFoundException();

    public TransactionNotFoundException() {
        super(ErrorCode.TRANSACTION_NOT_FOUND);
    }
}

File: src/main/kotlin/org/example/bankramenserver/domain/transaction/exception/TransactionNotFoundException.kt
package org.example.bankramenserver.domain.transaction.exception

import org.example.bankramenserver.global.error.exception.ErrorCode
import org.example.bankramenserver.global.error.exception.GlobalException

class TransactionNotFoundException : GlobalException(ErrorCode.TRANSACTION_NOT_FOUND) {
    companion object {
        @JvmField val EXCEPTION: GlobalException = TransactionNotFoundException()
    }
}
